tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "wit-widget",
+  "version": "0.1.4",
```

Comment:
Please add an instruction in RELEASE.md to update this version number as it gets serialized into JS payload (via jupyter/js/lib/index.js).

Reply:
done to the RELEASE.md in pip_package directory